.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - run:
           name: Install cljstyle CLI
           environment:
-            CLJSTYLE_VERSION: 0.13.0
+            CLJSTYLE_VERSION: 0.15.0
           command: |
             wget https://github.com/greglook/cljstyle/releases/download/${CLJSTYLE_VERSION}/cljstyle_${CLJSTYLE_VERSION}_linux.tar.gz
```

Comment:
Switch this to the `.zip` package while you're at it, the tarball will be discontinued in future releases.